apache/superset
# fix(Dashboard): Omnibar dropdown visibility and keyboard commands

### SUMMARY
This PR fixes a problem with the visibility of the Omnibar dropdown. It also implements the ability to close the Omnibar by clicking ESC. Finally, it removes the ability to open the Omnibar with the _ctrl/cmd + s_ in order to stop it from interfering with the default browser functionality. The Omnibar can now be opened with _ctrl/cmd + k_.

Fixes: #15539

### BEFORE

https://user-images.githubusercontent.com/60598000/128879772-97e45c5e-84a9-4996-af59-b085d9e963cd.mp4

### AFTER

https://user-images.githubusercontent.com/60598000/128880027-1694f0cb-de87-461b-9542-e1f72722dd08.mp4

### TESTING INSTRUCTIONS
1. Open a Dashboard
2. Press _cmd + k_ or _ctrl + k_
3. Observe the Omnibar appearing and search for a Dashboard
4. Make sure that the matching Dashboards appear
5. Press ESC and make sure the Omnibar closes

### ADDITIONAL INFORMATION


- [x] Has associated issue: #15539
